license.md

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2019 Mario Michel
```

Comment:
Please change `Mario Michel` to `apilayer GmbH`

Reply:
changed

Reply:
Please also squash your commits. Thank you.